spyder/utils/icon_manager.py

```diff
@@ -96,8 +96,10 @@
     'print':                   [('fa.print',), {'color': MAIN_FG_COLOR}],
     'fileclose':               [('fa.close',), {'color': MAIN_FG_COLOR}],
     'filecloseall':            [('fa.close', 'fa.close', 'fa.close'), {'options': [{'scale_factor': 0.6, 'offset': (0.3, -0.3), 'color': MAIN_FG_COLOR},  {'scale_factor': 0.6, 'offset': (-0.3, -0.3), 'color': MAIN_FG_COLOR}, {'scale_factor': 0.6, 'offset': (0.3, 0.3), 'color': MAIN_FG_COLOR}]}],
+    'breakpoint_transparent':  [('fa.circle',), {'color': 'rgba(139, 0, 0, 0.5)'}],  # darkred transparent
     'breakpoint_big':          [('fa.circle',), {'color': 'darkred'} ],
     'breakpoint_cond_big':     [('fa.question-circle',), {'color': 'darkred'},],
+    'arrow_debugger':          [('mdi.arrow-right-thick',), {'color': '#3775a9'}],
```

Comment:
Sorry @dalthviz, after seeing your last screenshot I think we should use `mdi.arrow-right-bold` instead because the arrow would be easier to locate visually.

Other than that, this looks great!